README.md

```diff
@@ -32,17 +32,17 @@ Or you can use `StrimziKafkaCluster` for a Kafka cluster with multiple Kafka bro
 
 ## How to use the Test container?
 
-Using a given test container takes three simple steps:
-1. Specify two environment variables
-2. Add the Strimzi test container as a Maven dependency
-3. Run Strimzi test container
+Using Strimzi test container takes two simple steps:
+1. Add the Strimzi test container as a Maven dependency
+2. Run Strimzi test container
     1. default configuration
     2. (optional) run Strimzi test container with additional configuration
 
-First, you need to specify two essential environment variables:
+You can specify environment variables to configure the container image to run:
```

Comment:
Did you rebase also `README.md`? It seems that you have a version with the `Env` stuff, which by this point is obsolete and we are no longer to use it :)

Reply:
It think they were there as I didn't have any conflicts. I am removing them